exporter/kinesisexporter/factory.go

```diff
@@ -44,7 +46,8 @@ func createDefaultConfig() configmodels.Exporter {
 			NameVal: typeStr,
 		},
 		AWS: AWSConfig{
-			Region: "us-west-2",
+			Region:     "us-west-2",
```

Comment:
Thought: Wouldn't this be better as `us-east-1` considering that is the oldest dc?

Reply:
It would be best to keep this as is for in case any other users aside from us expect this as the default behaviour